Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[workspace]
+members = [
+  "atk-sys-test",
+  "gdk-pixbuf-sys-test",
+  "gdk-sys-test",
+  "gio-sys-test",
+  "glib-sys-test",
+  "gobject-sys-test",
+  "gtk-sys-test",
+  "pango-sys-test",
+  "pangocairo-sys-test"
+]
```

Comment:
Can this be integrated with `cargo test --all` somehow?

Reply:
Should be possible by having an empty `main.rs` and having the current `main.rs` inside the `tests` directory.

Reply:
I removed workspaces, so there is no benefit from this now.